channels/az.m3u

```diff
@@ -1,25 +1,159 @@
 http://85.132.81.184:8080/atv-4/index.m3u8
+#EXTINF:-1 tvg-id="aznews.tv" tvg-name="AN.T TV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/BcVv8Hh.png" group-title="News",AZ NEWS TV (AN.T ) (720p)
+https://edge1.socialsmart.tv/aznews/smil/playlist.m3u8
+#EXTINF:-1 tvg-id="aznews.tv" tvg-name="AN.T TV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/BcVv8Hh.png" group-title="News",AZ NEWS TV (AN.T ) (720p)
+https://edge1.socialsmart.tv/aznews/smil/aznews/720p/chunks.m3u8
+#EXTINF:-1 tvg-id="aznews.tv" tvg-name="AN.T TV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/BcVv8Hh.png" group-title="News",AZ NEWS TV (AN.T ) (360p)
+https://edge1.socialsmart.tv/aznews/smil/aznews/360p/chunks.m3u8
+#EXTINF:-1 tvg-id="aznews.tv" tvg-name="AN.T TV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/BcVv8Hh.png" group-title="News",AZ NEWS TV (AN.T ) (240p)
+https://edge1.socialsmart.tv/aznews/smil/aznews/240p/chunks.m3u8
+#EXTINF:-1 tvg-id="azstartv.com" tvg-name="azstartv" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/ZDcu8he.png" group-title="",Az Star TV (1080p)
+http://live.livestreamtv.ca/azstar/smil:azstar.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="azstartv.com" tvg-name="azstartv" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/ZDcu8he.png" group-title="",Az Star TV (240p)
+http://live.livestreamtv.ca/azstar/smil:azstar.smil/chunklist_w1128625386_b400000_slen.m3u8
+#EXTINF:-1 tvg-id="aztv.az" tvg-name="AZ TV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="http://yoda.az/idmanlive/aztv_logo.png" group-title="",AZ TV (1080p)
+https://59c7ea6bec1c6.streamlock.net/live/aztv.stream/playlist.m3u8?hash=d0f574372c36bc44c833002f87764f1d
+#EXTINF:-1 tvg-id="aztv.az" tvg-name="AZ TV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="http://yoda.az/idmanlive/aztv_logo.png" group-title="",AZ TV (404p)
+http://str.yodacdn.net/aztv/tracks-v1a1/mono.m3u8?token=9cd1fe22fe5a1227a1de1ffc81b11d1f68a57f78-9e60c50db056a1ebab3444b946138d84-1618045991-1618035191
+#EXTINF:-1 tvg-id="baku.tv" tvg-name="BAKUTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/EesaTbO.png" group-title="News",BAKU TV (1080p)
+https://rtmp.baku.tv/streams/bakutv.m3u8
+#EXTINF:-1 tvg-id="baku.tv" tvg-name="BAKUTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/EesaTbO.png" group-title="News",BAKU TV (720p)
+https://rtmp.baku.tv/live/bakutv_720p.m3u8
+#EXTINF:-1 tvg-id="baku.tv" tvg-name="BAKUTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/EesaTbO.png" group-title="News",BAKU TV (540p)
+https://rtmp.baku.tv/live/bakutv_540p.m3u8
+#EXTINF:-1 tvg-id="baku.tv" tvg-name="BAKUTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/EesaTbO.png" group-title="News",BAKU TV (360p)
+https://rtmp.baku.tv/live/bakutv_360p.m3u8
```

Comment:
All of these links don't work.

Reply:
well all them work for me maybe if you want to let them there and add (kodi) tag to it ?

Reply:
Baku TV channels work for me, however AZ tv doesn't work.

![vlcsnap-2021-04-13-13h29m18s440](https://user-images.githubusercontent.com/30985701/114545448-47c5c600-9c5c-11eb-84e0-ced885ef7255.png)

Maybe not a 24/7 channel.

Reply:
Baku TV now works for me too. 

If the channel can sometimes not work, it is better to add the label `[not 24/7]` to its name.